go.mod

```diff
@@ -35,9 +35,9 @@ require (
 	github.com/soheilhy/cmux v0.1.5
 	github.com/spf13/cast v1.3.1
 	github.com/spf13/cobra v1.1.3
-	github.com/spf13/pflag v1.0.5
 	github.com/spf13/viper v1.7.1
 	github.com/stretchr/testify v1.7.0
+	github.com/tidwall/wal v0.1.4
```

Comment:
Try to avoid unstable (and not very actively maintained) dependencies

Reply:
AFAIK, this is the most popular general purpose Go WAL implementation and also last change in November 2020, which was 5 months ago, plus I know the author as a reliable person.

Reply:
The author @tidwall confirmed with me right now that the library is stable and he is a solid maintainer and knows his stuff.